doc/LanguageClient.txt

```diff
@@ -267,6 +267,20 @@ Signature: LanguageClient#Notify(method: String, params: Map | List)
 
 Send a notification to the current language server.
 
+*LanguageClient_contextMenu()*
+Signature: LanguageClient#contextMenu(...)
+
+Show list of all available actions.
+
+If optional dependency FZF is installed, actions will be displayed in a FZF
+prompt, selecting one of the action will then call the action's function.
+
+To skip FZF prompt even if FZF is installed and use native numbered list,
+add this variable to vimrc:
+`let g:LanguageClient_FzfContextMenu = 0`
```

Comment:
Please keep option names consistent, i.e., `..._fzfContextMenu`. Also, please add doc to this option.

Reply:
Fixed.